test/index.js

```diff
@@ -45,8 +47,10 @@ let testModules = [
   'version_management'
 ];
 
-if ((process.env.npm_config_NAPI_VERSION !== undefined) &&
-    (process.env.npm_config_NAPI_VERSION < 50000)) {
+const napiVersion = Number(process.env.npm_config_NAPI_VERSION || process.versions.napi);
```

Comment:
I think this will still disable testing the experimental tests by default right?  I think we want to make sure that if you just do a checkout that they will run unless you specifically do something to prevent that.  Otherwise It will be too easy for us to miss running the experimental tests when we land PRs.  Previously if npm_config_NAPI_VERSION was not set then all tests would run, afterwards the experimental tests will never run unless npm_config_NAPI_VERSION is set to run then....

Reply:
yeah, I noticed that with your mention.

Also I found out that napi_date related API was landed on Node.js v12 as experimental yet NAPI v5 on master. And bigint related API was landed on Node.js v8, v10, v12 and master as experimental. It might be not enough to control experimental features merely by NAPI_VERSION if we'd like to address failure of old Node.js version running node-addon-api tests locally with simple `npm test`.

So maybe NAPI_EXPERIMENTAL features should be filtered with Node.js major version.

Nevertheless, I'd like to submit a independent PR to address this issue. For this PR, we could enable travis for fast PR test check?

Reply:
Seems reasonable to me.  Will wait until Monday to see if we have any other comments, otherwise will land.